### PR TITLE
update metadata content to remove references to databases where relevant

### PR DESCRIPTION
**Pull request by shonaisbister-disco**

### CODE REVIEW

**Review by lucasdigital — APPROVED** [review]
I don't see any issues, providing we're happy with uncapitalised "hive" for consistency.

File: docs/connect-metastores.md [hunk 1]
```diff
@@ -4,19 +4,17 @@ title: Connect metastores for metadata migrations (preview)
 sidebar_label: Connect metastores
 ---
 
-The **1.8.3** version of LiveData Migrator includes a **preview** of a feature that migrates metadata between metastores. Metadata can be migrated from [Apache Hive](https://cwiki.apache.org/confluence/display/Hive/Home) and [AWS Glue](https://aws.amazon.com/glue/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc).
+The **1.8.3** version of LiveData Migrator includes a **preview** of a feature that migrates metadata. Metadata can be migrated from [Apache Hive](https://cwiki.apache.org/confluence/display/Hive/Home) and [AWS Glue](https://docs.aws.amazon.com/glue/latest/dg/populate-data-catalog.html).
```
Comment (benradstone) [inline docs/connect-metastores.md:7]: > Metadata can be migrated from [Apache Hive](https://cwiki.apache.org/confluence/display/Hive/Home) and [AWS Glue](https://docs.aws.amazon.com/glue/latest/dg/populate-data-catalog.html).

Is there any reason why we're so specific here? Surely we can migrate data from Azure SQL DB as well (and local filesystems).

Comment (shonaisbister-disco) [reply docs/connect-metastores.md:7]: Paul asked me to update it this way. He said (if I'm understanding my notes correctly) that metadata is always stored in an SQL database so it's weird to point it out like it was. I can't remember the reasoning behind removing local fs - is that somewhat questionable? - but will ask him again tonight.

File: docs/command-reference.md [hunk 2]
```diff
@@ -1391,7 +1391,7 @@ hive agent add filesystem --filesystem-id myfilesystem --root-folder /var/lib/my
 
 ### `hive agent add glue`
 
-Add an [AWS Glue](https://docs.aws.amazon.com/glue/latest/dg/what-is-glue.html) hive agent to connect to an [AWS-hosted database](https://docs.aws.amazon.com/whitepapers/latest/aws-overview/database.html) using the `hive agent add glue` command.
+Add an [AWS Glue](https://docs.aws.amazon.com/glue/latest/dg/populate-data-catalog.html) hive agent to connect to an [AWS-hosted database](https://docs.aws.amazon.com/whitepapers/latest/aws-overview/database.html) using the `hive agent add glue` command.
```
Comment (benradstone) [inline docs/command-reference.md:1394]: Change **[AWS-hosted database](https://docs.aws.amazon.com/whitepapers/latest/aws-overview/database.html)** to:

[AWS Glue data catalog](https://docs.aws.amazon.com/glue/latest/dg/populate-data-catalog.html)

I would then re-add the original hyperlink back in to the first mention of AWS Glue:

Add an [AWS Glue](https://docs.aws.amazon.com/glue/latest/dg/what-is-glue.html) hive agent

File: docs/connect-metastores.md (hunk 1 above)
Comment (benradstone) [reply docs/connect-metastores.md:7]: The local filesystem removal I can understand as it's essentially a dump of metadata that we do onto a filesystem (i.e. it's not very useful to anyone).

Just feels like we're not mentioning Azure when we should be.

File: docs/command-reference.md [hunk 3]
```diff
@@ -1391,7 +1391,7 @@ hive agent add filesystem --filesystem-id myfilesystem --root-folder /var/lib/my
...
 
 ```text title="Add AWS Glue agent"
 SYNOPSYS
```
Comment (benradstone) [inline docs/command-reference.md:1397]: Whilst you're here, can you change the following:

`s3:///test_bucket/` to `s3://test_bucket/`

It occurs twice in the Optional Parameters and Example for the hive agent add glue section.

File: docs/connect-metastores.md (hunk 1 above)
Comment (shonaisbister-disco) [reply docs/connect-metastores.md:7]: I'm going to leave it as it is for now, and if psm says to put it back tonight then will update it.

**Pull request merged**